code/game/g_active.c

```diff
@@ -490,7 +490,7 @@ static char *TimedMessage(void) {
     container_p c;
 
     if (!level.timedMessages->length) {
-        return "^1RPG-X ERROR: No messages to display";
+        return NULL;
```

Comment:
Should the return below also be changed?

Reply:
I think that if that second return is ever reached, something has been properly messed up and such a glaring message is probably warranted.